tmt/steps/provision/podman.py

```diff
@@ -295,6 +321,14 @@ def remove(self) -> None:
             self.podman(Command('container', 'rm', '-f', self.container))
             self.info('container', 'removed', 'green')
 
+        if self.network:
+            # will remove the network if there are no more containers attached to it
+            try:
+                self.podman(Command('network', 'rm', self.network))
+                self.info('container', 'network removed', 'green')
+            except tmt.utils.RunError:
+                pass
```

Comment:
I'm a bit worried about ignoring errors that are not as safe to ignore as "no such network". Would it be possible to check for the content of stderr, e.g. whether the "network not found" substring? We should re-raise the error otherwise, I suppose.

Reply:
handling this the same way as network create above now.